lib/routes/btbtla/detail.ts

```diff
@@ -0,0 +1,95 @@
+import { load } from 'cheerio'; // 类似 jQuery 的 API HTML 解析器
+
+import type { Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+
+export const route: Route = {
+    path: '/detail/:name',
+    categories: ['multimedia'],
+    example: '/btbtla/detail/雍正王朝',
+    parameters: { name: '电影 | 电视剧名称' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: true,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'BTBTLA | 指定剧名',
+    maintainers: ['Hermes1030'],
+    handler,
+};
+
+async function handler(ctx) {
+    let name = '';
+    if (ctx.req.path.includes('/detail/')) {
+        name = ctx.req.path.split('/detail/')[1].split('.')[0];
+    }
```

Comment:
[Use param()](https://hono.dev/docs/api/request#param) to extract path parameter